scripts/run_pipeline.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
It looks like you're using `-` instead of `_` everywhere else in the file names. Should it be `run-pipeline.sh`?